src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java

```diff
@@ -1428,7 +1428,6 @@ void shouldUpdateRespondentAndApplicantWithPartyNameAndPartyTypeDisplayValue_whe
         @Test
         void shouldAssignCaseName1v2_whenCaseIs1v2GlobalSearchEnabled() {
             // Given
```

Comment:
Remove GlobalSerach int name as not relevant
ie shouldAssignCaseName1v2_whenCaseIs1v2()

Reply:
Test name updated